src/pal/src/exception/seh-unwind.cpp

```diff
@@ -195,6 +198,9 @@ BOOL PAL_VirtualUnwind(CONTEXT *context, KNONVOLATILE_CONTEXT_POINTERS *contextP
 
     return TRUE;
 }
+#else
+#error don't know how to unwind on this platform
```

Comment:
Shouldn't CMake catch the lack of libunwind and fail to generate makefiles? Seems like that's the right place for the error message.